install_sct.bat

```diff
@@ -187,7 +187,7 @@ python\envs\venv_sct\Scripts\pip install -e . || goto error
 rem Install external dependencies
 echo:
 echo ### Downloading model files and binaries...
-FOR %%D IN (PAM50 deepseg_sc_models deepseg_gm_models deepseg_lesion_models binaries_win deepreg_models PAM50_normalized_metrics) DO sct_download_data -d %%D -k || goto error
+FOR %%D IN (PAM50 deepseg_sc_models deepseg_gm_models deepseg_lesion_models binaries_win deepreg_models PAM50_normalized_metrics) DO sct_download_data -d %%D -k || echo Failed to install '%%D'. Please re-run 'sct_download_data -d %%D' after installation.

```

Comment:
I want to say "ditto" here, but honestly I have no idea how to do even this much string manipulation in `.bat` scripts.

Reply:
I guess if we push this check to `sct_check_dependencies`, then that takes care of the problem for us!